manual/de/Help_ch_Tools_Assistants.xml

```diff
@@ -4188,136 +4188,189 @@
                 </para>
               </listitem>
             </varlistentry>
 
             <varlistentry>
-              <term><guilabel>Reminder</guilabel></term>
+              <term>Erinnerung</term>
 
               <listitem>
-                <para>You are <emphasis>n</emphasis> days from the scheduled transaction due date. The number of days
-                  <emphasis>n</emphasis> to remind in advance, is set either in the
-                  <guilabel>&appname; Preferences</guilabel> <link linkend="prefs-sched">Scheduled
-                  Transactions</link> tab or in the <guilabel>Overview</guilabel> tab of the
-                  <link linkend="sched-editor">Scheduled Transaction Editor</link>.
+                <para>Es verbleiben noch <emphasis>n</emphasis> Tage, bevor die geplante Buchung fällig wird. Die Anzahl
+                  der Tage <emphasis>n</emphasis> an die im Voraus erinnert werden soll, wird
+                  entweder in den <guilabel>&appname; Einstellungen</guilabel> unter
+                  <xref linkend="prefs-sched" /> oder auf der Registerkarte
+                  <guilabel>Übersicht</guilabel> in dem <xref linkend="sched-editor" />
+                  angegeben.
                 </para>
               </listitem>
             </varlistentry>
 
             <varlistentry>
-              <term><guilabel>To-create</guilabel></term>
+              <term>Zu Erstellen</term>
 
               <listitem>
-                <para>This scheduled transaction will be automatically created when you press <guibutton>OK</guibutton>.
+                <para>Diese terminierte Buchung wird automatisch erstellt, wenn Sie <guibutton>OK</guibutton> drücken.
                 </para>
               </listitem>
             </varlistentry>
 
             <varlistentry>
-              <term><emphasis>Empty</emphasis></term>
+              <term>Leer</term>
 
               <listitem>
-                <para>An empty field means that no operations are pending for the scheduled transaction.
+                <para>Wird eine Buchung aufgelistet, für die kein Ausführungsdatum angezeigt wird, bedeutet das,
+                  dass eine terminierte Buchung im <xref linkend="sched-editor" /> angelegt ist, allerdings momentan
+                  keine Vorgänge zur Ausführung anstehen.
                 </para>
               </listitem>
             </varlistentry>
           </variablelist>
-        </listitem>
-      </varlistentry>
-    </variablelist>
+        </glossdef>
+      </glossentry>
+<!-- FIXME source: new item -->
+      <glossentry>
+        <glossterm>Wert</glossterm>
+
+        <glossdef>
+          <para>Ist für die automatische Erstellung noch die Angabe des Buchungswerts notwendig, so wird das mit
+            dem Hinweis <emphasis>(Wert benötigt)</emphasis> angezeigt. Dies kann z.B. bei einer
+            Buchung zwischen zwei Konten mit unterschiedlichen Währungen notwendig sein.
```

Comment:
Ob eine Dividende oder andere Gewinnausschüttung und in welche Höhe gezahlt wird, wird ja reativ kurzfristig festgelegt. Ebenso können Kursschwankungen eine Rolle spielen.